src/core_write.cpp

```diff
@@ -17,12 +17,16 @@
 
 UniValue ValueFromAmount(const CAmount& amount)
 {
-    bool sign = amount < 0;
-    int64_t n_abs = (sign ? -amount : amount);
-    int64_t quotient = n_abs / COIN;
-    int64_t remainder = n_abs % COIN;
+    int64_t quotient = amount / COIN;
+    if (quotient < 0) {
+        quotient = -quotient;
```

Comment:
It kind of feels weird to me how quotient and remainder are treated as positive/negative separately here. It's only ever possible for either both of them to be positive (defining positive as >=0), or both of them to be negative, right?

Reply:
Good points. Feedback addressed!